crt_portal/cts_forms/templates/forms/confirmation.html

```diff
@@ -3,15 +3,15 @@
 {% load static %}
 
 {% block page_header %}
-<header class="confirmation-header">
```

Comment:
There were a few duplicative classes like this that implemented existing type rules under different class names. I simplified where it seemed to make sense.